src/resources/views/crud/fields/browse_multiple.blade.php

```diff
@@ -97,7 +97,11 @@
 
             // function to use the files selected inside elfinder
             function processSelectedMultipleFiles(files, requestingField) {
-                elfinderTarget.trigger('createInputsForItemsSelectedWithElfinder', [files]);                
+                files = files.map(function(file){
+                    file.path = file.path.replace(/\\/g,"/");
+                    return file;
+                });
```

Comment:
Guys, what do you think about moving this inside the `createInputsForItemsSelectedWithElfinder` event handler? I think that'd be a bit cleaner, right? If that's the only place we use it.

Reply:
I agree, so basically make a function and then call that function inside the `createInputsForItemsSelectedWithElfinder` event handlers, since there are two definition for the event handler based on the `multiple` parameter.

Though I'm wondering why is there `multiple` param for the `browse_multiple` field. I mean, the field name says itself.

Waiting for others input